test/performance/array/distCreate-arrays-deinit.cc-perf.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: arrDeinit-GETS:, arrDeinit-PUTS:, arrDeinit-ONS:, arrDeinit-GETS:, arrDeinit-PUTS:, arrDeinit-ONS:, arrDeinit-GETS:, arrDeinit-PUTS:, arrDeinit-ONS:
+graphkeys: blockArrDeinit-GETS:, blockArrDeinit-PUTS:, blockArrDeinit-ONS:, cyclicArrDeinit-GETS:, cyclicArrDeinit-PUTS:, cyclicArrDeinit-ONS:, blockCycArrDeinit-GETS:, blockCycArrDeinit-PUTS:, blockCycArrDeinit-ONS:
+repeat-files: distCreate-cc-block.dat, distCreate-cc-cyclic.dat, distCreate-cc-blockCyc.dat
+ylabel: Count
+graphtitle: Destroying distributed arrays
```

Comment:
Graph title is the same for each graph currently

Reply:
I don't think so?

Reply:
Yeah, my mistake